portfolio/src/main/webapp/index.html

```diff
@@ -44,5 +44,7 @@ <h1>My Portfolio</h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+    <button onclick="getBookInfo();">Click to run getBookInfo()</button>
```

Comment:
Moved the comment from onload to a button for testing purposes. Later impl will have onload call a different function.